packages/fx-core/src/common/azure-hosting/azureFunctionHosting.ts

```diff
@@ -3,6 +3,10 @@
 
 import { AzureHosting } from "./azureHosting";
 import { HostType } from "./interfaces";
+import { Inputs, TokenProvider } from "@microsoft/teamsfx-api";
+import { Void } from "../../plugins";
+import { AzureOperations } from "../../plugins/resource/bot/azureOps";
+import { azureWebSiteDeploy } from "./utils";
 
 const functionResourceId = "provisionOutputs.functionOutput.value.resourceId";
```

Comment:
@hund030 Same for the class name, we need to use "AzureFunctionsHosting" as branding name (has "s").